services/user/Chainmail/ui/package.json

```diff
@@ -41,6 +46,7 @@
         "class-variance-authority": "^0.7.0",
         "clsx": "^2.1.1",
         "date-fns": "^3.6.0",
+        "element-internals-polyfill": "^0.1.49",
```

Comment:
I see new dependencies introduced here: `element-internals-polyfill` and the `@codemirror/*` dependencies. Were these required by another updated package?

Reply:
I added everything that yarn complained about as "missing peer dependency"

Reply:
Is there a better way to handle that?

Reply:
This is fine here. Doesn't affect build/bundle size much at all. And gets rid of the warnings.